lexicons/app/bsky/actor/createScene.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.actor.createScene",
+  "type": "procedure",
+  "description": "Create a scene.",
+  "parameters": {},
+  "input": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": ["handle"],
+      "properties": {
+        "handle": {"type": "string"},
+        "recoveryKey": {"type": "string"}
+      }
+    }
+  },
+  "output": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": ["handle", "did", "declarationCid"],
+      "properties": {
+        "handle": { "type": "string" },
+        "did": { "type": "string" },
+        "declarationCid": { "type": "string" }
+      }
+    }
+  },
+  "errors": [
+    {"name": "HandleNotAvailable"}
+  ]
```

Comment:
Could add `InvalidHandle` here if we're feeling it, not sure that it's really critical though.

Reply:
yeah good catch 👍